src/cascadia/TerminalApp/App.xaml

```diff
@@ -93,6 +93,21 @@
                                             ResourceKey="ApplicationPageBackgroundThemeBrush" />
                         </ResourceDictionary>
 
+                        <ResourceDictionary x:Key="HighContrast">
+                            <!--  Define resources for HighContrast mode here  -->
+
+                            <!--
+                                MSFT:38264744
+                                You'd think that if a key wasn't present in a ThemeDictionary,
+                                it'd fall back to the original value. You'd be wrong - if you
+                                provide a Light&dark version of a resource, but not the
+                                HighContrast version, the resource loader will fall back to the
+                                Light value.
+                            -->
+                            <StaticResource x:Key="TabViewBackground"
+                                            ResourceKey="SystemColorButtonFaceColorBrush" />

```

Comment:
It might be worthwhile to note that `SystemColorButtonFaceColorBrush` is the default background color for WinUI's `TabViewBackground` under high contrast mode.
Otherwise it's a bit perplexing why we'd use the color of a button _face_ (foreground?) to draw the background of a tab switcher...

Reply:
I'm okay to merge without this comment fwiw but if you want to add it @zadjii-msft I'll hold for 10